pkg/controller/backup_finalizer_controller.go

```diff
@@ -150,8 +157,14 @@ func (r *backupFinalizerReconciler) Reconcile(ctx context.Context, req ctrl.Requ
 		SkippedPVTracker: pkgbackup.NewSkipPVTracker(),
 	}
 	var outBackupFile *os.File
+	var volumeInfos []*volume.VolumeInfo
 	if len(operations) > 0 {
-		// Call itemBackupper.BackupItem for the list of items updated by async operations
+		volumeInfos, err = backupStore.GetBackupVolumeInfos(backup.Name)
```

Comment:
Let's move this `GetBackupVolumeInfos` to `updateVolumeInfos@backup.go`, so that the major functions for async operation finalizing are decoupled from the specific requirements.

Reply:
Modified.